src/training_data.py

```diff
@@ -86,12 +95,32 @@ def load_luis_data(self,data):
                 #print(text[start:end])
                 entities.append({"entity":e["entity"],"value":val,"start":start,"end":end})
 
-            self.intent_examples.append({"text":text,"intent":intent})            
+            self.intent_examples.append({"text":text,"intent":intent})
             self.entity_examples.append({"text":text,"intent":intent,"entities":entities})
-                            
-                
-            
-    def load_data(self,data):
+
+    def load_api_data(self, files):
+        for filename in files:
+            data = json.loads(open(filename,'rb').read())
+            # get only intents, skip the rest. The property name is the target class
+            if "userSays" not in data:
+                continue
+
+            intent = data.get("name")
+            for s in data["userSays"]:
+                text = unicode("".join(map(lambda chunk: chunk["text"], s.get("data"))))
+                # add entities to each token, if available
+                entities = []
+                for e in filter(lambda chunk: "alias" in chunk or "meta" in chunk, s.get("data")):
+                    val = u"\s*" + unicode(e["text"])
```

Comment:
@plauto this unicode hacking is a bit dirty and I think we can avoid it :) 

I would propose we read the json files like this
```python
import codecs
f = codecs.open('data.json', encoding='utf-8')
data = json.loads(f.read())
```
then we should only have unicode objects everywhere and we can avoid using any `str` at all. NB this may have downstream effects in e.g. `spacy_tokenizer.py` so we should run all the tests to check everything still works.